lighthouse/src/main.rs

```diff
@@ -43,12 +45,11 @@ fn main() {
             Arg::with_name("spec")
                 .short("s")
                 .long("spec")
-                .value_name("TITLE")
-                .help("Specifies the default eth2 spec type.")
+                .value_name("DEPRECATED")
+                .help("This flag is deprecated, it will be disallowed in a future release. This
+                    value is now derived from the --testnet or --testnet-dir flags.")
```

Comment:
I think for multiline strings you want to end the line with `_\` (space backslash) so that clap can do the wrapping and it doesn't look weird. This is what it looks like rn:

```
    -s, --spec <DEPRECATED>
            This flag is deprecated, it will be disallowed in a future release. This
                                value is now derived from the --testnet or --testnet-dir flags.
```